apps/contentful-apps/proxy.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "/api": {
+    "target": "http://localhost:4444",
+    "secure": false
+  }
+}
```

Comment:
This app doesn't need to proxy any api stuff, right? Can remove this and from the configuration/server.ts

Reply:
Ok, I've removed the proxy.config.json and also taken it out of the server.ts file.